Quorum/llm/prompts/first_deposit_prompt.j2

```diff
@@ -0,0 +1,24 @@
+You are an expert Solidity code analyzer specialized in proposals. You will be provided with a snippet of Solidity source code representing a proposal payload that contains new assets listing logic.
```

Comment:
lets change it to "governance proposals"

Reply:
For now its working in the regression test, so if you encounter proposal when it doesn't work as expected, let me know and we can adopt the prompt accordingly